static/styles/widgets.scss

```diff
@@ -90,18 +90,18 @@ button.add-task,
 button.poll-comment,
 button.poll-question {
     border-radius: 3px;
-    border: 1px solid #cccccc;
+    border: 1px solid hsl(0, 0%, 80%);
     background-color: white;
     width: 100px;
 }
 
 button.add-task:hover,
 button.poll-comment:hover,
 button.poll-question:hover {
-    border-color: #999999;
+    border-color: hsl(0, 0%, 60%);
 }
 
 .widget-error {
-    color: #b94a48;
+    color: hsl(1, 45%, 50%);
```

Comment:
https://github.com/zulip/zulip/commit/6e9bf1b5634d78ad2d7db4f51#diff-1945408b3b614123b58664589a3fc4f3R209

`color: hsl(1, 44%, 50%);`

Reply:
Looks like the color for the widget-error changed in the codebase